app/script/components/groupVideoGrid.js

```diff
@@ -77,9 +90,12 @@ z.components.GroupVideoGrid = class GroupVideoGrid {
     });
   }
 
-  doubleClickedOnVideo(data, event) {
-    const childVideo = event.currentTarget.querySelector('video');
-    childVideo.classList.toggle(GroupVideoGrid.CONFIG.CONTAIN_CLASS);
+  doubleClickedOnVideo(_, {currentTarget}) {
```

Comment:
Please keep a proper naming here. Even the linter should complain. First param is the view model in knockout.